<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="18" passed="16" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-14T17:04:37 MSK" name="Surefire suite" finished-at="2022-10-14T17:06:06 MSK" duration-ms="88532">
    <groups>
    </groups>
    <test started-at="2022-10-14T17:04:37 MSK" name="Surefire test" finished-at="2022-10-14T17:06:06 MSK" duration-ms="88532">
      <class name="LoginCourierTest">
        <test-method signature="tryAuthCourierWithoutLoginReturnBadRequest()" started-at="2022-10-14T17:04:51 MSK" name="tryAuthCourierWithoutLoginReturnBadRequest" finished-at="2022-10-14T17:04:51 MSK" duration-ms="292" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryAuthCourierWithoutLoginReturnBadRequest -->
        <test-method signature="tryAuthCourierIncorrectLoginThenNotFound()" started-at="2022-10-14T17:04:51 MSK" name="tryAuthCourierIncorrectLoginThenNotFound" finished-at="2022-10-14T17:04:52 MSK" duration-ms="1418" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryAuthCourierIncorrectLoginThenNotFound -->
        <test-method signature="authCourierPositiveTest()" started-at="2022-10-14T17:04:51 MSK" name="authCourierPositiveTest" finished-at="2022-10-14T17:05:54 MSK" duration-ms="63114" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- authCourierPositiveTest -->
        <test-method signature="tryAuthCourierWithoutPasswordReturnBadRequest()" started-at="2022-10-14T17:04:51 MSK" name="tryAuthCourierWithoutPasswordReturnBadRequest" finished-at="2022-10-14T17:05:53 MSK" duration-ms="61784" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <400> but was <504>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <504>.

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at jdk.internal.reflect.GeneratedMethodAccessor137.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at ru.yandex.practikum.utils.Utils.checkResponseReturnBadRequestCode(Utils.java:21)
at LoginCourierTest.tryAuthCourierWithoutPasswordReturnBadRequest(LoginCourierTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at org.testng.junit.JUnit4TestRunner.start(JUnit4TestRunner.java:71)
at org.testng.junit.JUnit4TestRunner.run(JUnit4TestRunner.java:56)
at org.testng.TestRunner$1.run(TestRunner.java:679)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.runJUnitWorkers(TestRunner.java:885)
at org.testng.TestRunner.privateRunJUnit(TestRunner.java:708)
at org.testng.TestRunner.run(TestRunner.java:599)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:198)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryAuthCourierWithoutPasswordReturnBadRequest -->
        <test-method signature="tryAuthNonexistentCourierThenNotFound()" started-at="2022-10-14T17:04:51 MSK" name="tryAuthNonexistentCourierThenNotFound" finished-at="2022-10-14T17:05:55 MSK" duration-ms="64316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryAuthNonexistentCourierThenNotFound -->
        <test-method signature="tryAuthCourierIncorrectPasswordThenNotFound()" started-at="2022-10-14T17:04:51 MSK" name="tryAuthCourierIncorrectPasswordThenNotFound" finished-at="2022-10-14T17:05:56 MSK" duration-ms="65471" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryAuthCourierIncorrectPasswordThenNotFound -->
      </class> <!-- LoginCourierTest -->
      <class name="CreateNewCourierTest">
        <test-method signature="tryCreateDuplicateLoginCourierThenConflict()" started-at="2022-10-14T17:04:38 MSK" name="tryCreateDuplicateLoginCourierThenConflict" finished-at="2022-10-14T17:04:47 MSK" duration-ms="9565" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryCreateDuplicateLoginCourierThenConflict -->
        <test-method signature="tryCreateDuplicateCourierThenConflict()" started-at="2022-10-14T17:04:38 MSK" name="tryCreateDuplicateCourierThenConflict" finished-at="2022-10-14T17:04:48 MSK" duration-ms="10808" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryCreateDuplicateCourierThenConflict -->
        <test-method signature="createCourierWithoutFirstNamePositiveTest()" started-at="2022-10-14T17:04:38 MSK" name="createCourierWithoutFirstNamePositiveTest" finished-at="2022-10-14T17:04:49 MSK" duration-ms="11793" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCourierWithoutFirstNamePositiveTest -->
        <test-method signature="createCourierPositiveTest()" started-at="2022-10-14T17:04:38 MSK" name="createCourierPositiveTest" finished-at="2022-10-14T17:04:50 MSK" duration-ms="12673" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createCourierPositiveTest -->
        <test-method signature="tryCreateCourierWithoutLoginReturnBadRequest()" started-at="2022-10-14T17:04:38 MSK" name="tryCreateCourierWithoutLoginReturnBadRequest" finished-at="2022-10-14T17:04:51 MSK" duration-ms="12956" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryCreateCourierWithoutLoginReturnBadRequest -->
        <test-method signature="tryCreateCourierWithoutPasswordReturnBadRequest()" started-at="2022-10-14T17:04:38 MSK" name="tryCreateCourierWithoutPasswordReturnBadRequest" finished-at="2022-10-14T17:04:51 MSK" duration-ms="13263" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tryCreateCourierWithoutPasswordReturnBadRequest -->
      </class> <!-- CreateNewCourierTest -->
      <class name="OrderParamTest">
        <test-method signature="placeOrderDifferentColorsPositiveTest()" started-at="2022-10-14T17:06:03 MSK" name="placeOrderDifferentColorsPositiveTest[0]" finished-at="2022-10-14T17:06:04 MSK" duration-ms="645" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- placeOrderDifferentColorsPositiveTest[0] -->
        <test-method signature="placeOrderDifferentColorsPositiveTest()" started-at="2022-10-14T17:06:03 MSK" name="placeOrderDifferentColorsPositiveTest[3]" finished-at="2022-10-14T17:06:06 MSK" duration-ms="2579" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- placeOrderDifferentColorsPositiveTest[3] -->
        <test-method signature="placeOrderDifferentColorsPositiveTest()" started-at="2022-10-14T17:06:03 MSK" name="placeOrderDifferentColorsPositiveTest[1]" finished-at="2022-10-14T17:06:05 MSK" duration-ms="1266" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- placeOrderDifferentColorsPositiveTest[1] -->
        <test-method signature="placeOrderDifferentColorsPositiveTest()" started-at="2022-10-14T17:06:03 MSK" name="placeOrderDifferentColorsPositiveTest[2]" finished-at="2022-10-14T17:06:05 MSK" duration-ms="1921" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- placeOrderDifferentColorsPositiveTest[2] -->
      </class> <!-- OrderParamTest -->
      <class name="OrderListTest">
        <test-method signature="getTotalOrderListNotNull()" started-at="2022-10-14T17:05:56 MSK" name="getTotalOrderListNotNull" finished-at="2022-10-14T17:06:03 MSK" duration-ms="6894" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTotalOrderListNotNull -->
        <test-method signature="getOrderListForCreatedCourierNotNull()" started-at="2022-10-14T17:05:56 MSK" name="getOrderListForCreatedCourierNotNull" finished-at="2022-10-14T17:05:59 MSK" duration-ms="3136" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <200> but was <404>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at jdk.internal.reflect.GeneratedMethodAccessor137.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at jdk.internal.reflect.GeneratedMethodAccessor168.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at ru.yandex.practikum.utils.Utils.checkResponseReturnSuccessCode(Utils.java:13)
at OrderListTest.getOrdersCount(OrderListTest.java:87)
at OrderListTest.getOrderListForCreatedCourierNotNull(OrderListTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at org.testng.junit.JUnit4TestRunner.start(JUnit4TestRunner.java:71)
at org.testng.junit.JUnit4TestRunner.run(JUnit4TestRunner.java:56)
at org.testng.TestRunner$1.run(TestRunner.java:679)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.runJUnitWorkers(TestRunner.java:885)
at org.testng.TestRunner.privateRunJUnit(TestRunner.java:708)
at org.testng.TestRunner.run(TestRunner.java:599)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:198)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getOrderListForCreatedCourierNotNull -->
      </class> <!-- OrderListTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
